cloudpickle/cloudpickle.py

```diff
@@ -1268,3 +1276,11 @@ def _typevar_reduce(obj):
     if module_and_name is None:
         return (_make_typevar, _decompose_typevar(obj))
     return (getattr, module_and_name)
+
+
+def _get_bases(typ):
+    if hasattr(typ, '__orig_bases__'):
+        bases_attr = '__orig_bases__'
```

Comment:
maybe add an inline comment such as:
"PEP 560 support for pickling dynamic classes subclassing `Generic` type annotations"

Reply:
I did that as a suggestion in my review below.